src/main/java/server/Router.java

```diff
@@ -0,0 +1,19 @@
+package main.java.server;
+
+import main.java.server.request.Request;
+
+import java.util.HashMap;
+
+public class Router {
+
+    public String route(Request request) {
+        HashMap<String, String> requestHeaders = request.getHeaders();
+        String path = requestHeaders.get("path");
+        if (path.equals("/simple_get")) {
+            return "HTTP/1.1 200 OK\r\n";
```

Comment:
Similarly, I imagine this string might see some reuse - at which point it could be helpful to have it wrapped in a variable or constant.